plugins/utils/mini.nix

```
@@ -1,27 +1,24 @@
 {
   lib,
-  helpers,
-  config,
   pkgs,
   ...
 }:
-with lib;
-let
-  cfg = config.plugins.mini;
-in
-{
-  options.plugins.mini = {
-    enable = mkEnableOption "mini.nvim";
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "mini";
+  originalName = "mini-nvim";
 
-    package = lib.mkPackageOption pkgs "mini.nvim" {
-      default = [
-        "vimPlugins"
-        "mini-nvim"
-      ];
-    };
+  maintainers = [ lib.maintainers.khaneliman ];
 
-    modules = mkOption {
-      type = with types; attrsOf attrs;
+  package = lib.mkPackageOption pkgs "mini.nvim" {
+    default = [
+      "vimPlugins"
+      "mini-nvim"
+    ];
+  };
+
+  extraOptions = {
+    modules = lib.mkOption {
+      type = with lib.types; attrsOf attrs;
```

```suggestion
      type = with lib.types; attrsOf (attrsOf anything);
```

`attrs` doesn't merge recursively and is therefore deprecated